packages/theme/src/theme/_buttons.scss

```diff
@@ -76,6 +76,43 @@
 		padding: 0 $padding-smaller;
 	}
 
+	&-active-icon {
+		background-color: unset;
+		border: 1px solid #333;
+		border-radius: 12px;
+		box-shadow: none;
+		color: #333;
+		height: 24px;
+		line-height: unset;
+		opacity: 0.5;
+		padding: 0;
+		width: 24px;
+
+		&:hover,
+		&:active,
+		&:focus {
+			background-color: unset;
+			box-shadow: none;
+		}
+
+		&:hover {
+			opacity: 0.75;
+		}
+
+		svg {
+			height: 14px;
+			margin: 4px;
+			width: 14px;
+		}
+
+		&.active {
+			background-color: $brand-primary;
+			border: 1px solid $brand-primary;
```

Comment:
border is already set, please change only the border color
unless this is to override existing things ?